src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -36,4 +37,15 @@ class Bic extends Constraint
     );
 
     public $message = 'This is not a valid Business Identifier Code (BIC).';
+
+    public function __construct($options = null)
+    {
+        // @deprecated since Symfony 4.2
+        // throw new LoginException(sprintf('The "symfony/intl" component is required to use the "%s" constraint.', __CLASS__));
```

Comment:
should be inside the "if", where it will be in 5 :) (no need for L43 to me)

Reply:
fixed